spyql/processor.py

```diff
@@ -393,9 +427,34 @@ def _go(self, output_handler):
         return input_row_number - (1 if self.has_header else 0)
 
 
+class InteractiveProcessor(Processor):
+    def __init__(self, prs, strings, source = None):
+        super().__init__(prs, strings, interactive = True)
+        self.source = source
+
+    def get_input_iterator(self):
+        # _from = self.prs["from"] # now is "PYTHON"
+        data = self.vars["user_query_vars"]
+        if self.source != None:
+            if self.source in data:
+                spyql.log.user_info(f"Trying to read from python object")
+                return data[self.source]
+            else:
+                spyql.log.user_info(f"Trying to read as python expression")
+                e = self.eval_clause("from", self.compile_clause("from"))
+                if e:
+                    if not isiterable(e):
+                        e = [e]
+                    if not isiterable(e[0]):
+                        e = [[el] for el in e]
+                return e
+        else:
+            return self.prs["select"]
+
+
```

Comment:
I think we do not need this class, and it's actual very redundant vs `PythonExprProcessor`. 
But please assess.

Reply:
Removed